### PR TITLE
Update base-ru.yaml

### PR DESCRIPTION
**Pull request by L404-Venik**
Hello. I am russian student and I'd like to help you with game translation. I changed some things in the beginning just for example.

### CODE REVIEW

**Comment by CubasticBanjo** [comment]
not so good

**Comment by L404-Venik** [comment]
Wait, I'm confused. did you like it? Because I got email that it's: "not so good". I did just couple sentences to try. Should I continue?

File: translations/base-ru.yaml [hunk 1]
```diff
@@ -3,12 +3,13 @@ steamPage:
         создания и объединения все более сложных фигур на бесконечной карте.
     discordLinkShort: Официальный Discord сервер
     intro: >-
-        Shapez.io - это спокойная игра о создании фабрик для автоматизации
-        создания сложных геометрических фигур.
-
+        Shapez.io - Любите игры про автоматизацию? Тогда вы по адресу.
+
```
Comment (ZrubekLukas) [inline translations/base-ru.yaml:7]: Please remove trailing spaces on line 7 to get the lint check pass.

**Comment by ZrubekLukas** [comment]
Line 7 looks empty, but there are spaces on that line. These should be deleted. Empty line can stay but should be really empty. What you did is that you combined lines 8 and 9. Now this line is too long. See the result of yaml-lint check yourself.

**Comment by L404-Venik** [comment]
Finally.  Sorry for my stupidity. Now I realised what's were the problem.

**Comment by tobspr** [comment]
Ty!

**Pull request merged**